src/instruments/src/EFB/Localization/de.json

```diff
@@ -149,6 +149,7 @@
     },
     "Payload": {
       "BoardingTime": "Boarding-Dauer",
+      "EstimatedDurationUnit": "minuten",
```

Comment:
You do not have to provide other translations - this will get overwritten by our import from Localazy anyway. 
Updating the en.json is good as this is the source language I use to manually add new keys.

Reply:
Should I undo it?

Reply:
No, that's ok. We can use this one as well :)

Reply:
> Should I undo it?

You can - but no need - will be overwritten anyway.

Reply:
Okay, thanks!